src/fastga/models/aerodynamics/components/compute_cl_aileron.py

```diff
@@ -42,14 +42,16 @@ def initialize(self):
     # pylint: disable=missing-function-docstring
     # Overriding OpenMDAO setup
     def setup(self):
+        speed = "_low_speed" if self.options["low_speed_aero"] else "cruise"
         self.add_subsystem(
-            name="aileron_alpha",
+            name="aileron_alpha" + speed,
```

Comment:
Any reason for that change ? And if you want to keep it, for consistency I would call it `ls_tag` instead of `speed`